curp/src/client.rs

```diff
@@ -364,13 +368,19 @@ where
                     return Err(CommandProposeError::AfterSync(e));
                 }
             }
+            retry_timeout.next_retry();
         }
         Err(CommandProposeError::Propose(ProposeError::Timeout))
     }
 
     /// The shutdown rpc of curp protocol
     #[instrument(skip_all)]
     pub async fn shutdown(&self) -> Result<(), ProposeError> {
+        let mut retry_timeout = BackOff::new(
+            *self.config.initial_retry_timeout(),
```

Comment:
could we cache a `BackOff` and clone to reuse it?

Reply:
I think individual request should start from the initial timeout.  Why do we need to cache it?